custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,247 @@
+        return HVAC_MODE_FAN_ONLY
+
+    @property
+    def hvac_modes(self) -> List[str]:
+        """Return the list of available hvac operation modes."""
+        return [HVAC_MODE_FAN_ONLY]
+
+    async def async_set_hvac_mode(self, hvac_mode: str) -> None:
+        """Not implemented since there is only one hvac_mode."""
+
+    @property
+    def preset_mode(self) -> Optional[str]:
+        """Return the current preset mode, e.g., auto, smart, interval, favorite."""
+        state_ventilation_configuration = self.select_state(
+            IO_VENTILATION_CONFIGURATION_MODE_STATE
+        )
+        state_ventilation_mode = self.select_state(IO_VENTILATION_MODE_STATE)
+        state_prog = state_ventilation_mode.get("prog")
+
+        if state_prog == "on":
+            return PRESET_PROG
+
+        if state_ventilation_configuration == "comfort":
+            return PRESET_AUTO
+
+        if state_ventilation_configuration == "standard":
+            return PRESET_MANUAL
+
+        return None
+
+    @property
+    def preset_modes(self) -> Optional[List[str]]:
+        """Return a list of available preset modes."""
+        return PRESET_MODES
+
+    async def async_set_preset_mode(self, preset_mode: str) -> None:
+        """Set the preset mode of the fan."""
+        if preset_mode == PRESET_AUTO:
+            await self.async_execute_command(
+                COMMAND_SET_VENTILATION_CONFIGURATION_MODE, "comfort"
+            )
+            await self._set_ventilation_mode(prog="off")
+
+        if preset_mode == PRESET_PROG:
+            await self.async_execute_command(
+                COMMAND_SET_VENTILATION_CONFIGURATION_MODE, "standard"
+            )
+            await self._set_ventilation_mode(prog="on")
+
+        if preset_mode == PRESET_MANUAL:
+            await self.async_execute_command(
+                COMMAND_SET_VENTILATION_CONFIGURATION_MODE, "standard"
+            )
+            await self._set_ventilation_mode(prog="off")
+
+        await self.async_execute_command(
+            COMMAND_REFRESH_VENTILATION_STATE,
+        )
+        await self.async_execute_command(
+            COMMAND_REFRESH_VENTILATION_CONFIGURATION_MODE,
+        )
+
+    @property
+    def fan_mode(self) -> Optional[str]:
+        """Return the fan setting."""
+        ventilation_mode_state = self.select_state(IO_VENTILATION_MODE_STATE)
+        cooling = ventilation_mode_state.get("cooling")
+
+        if cooling == "on":
+            return FAN_BYPASS
+        else:
+            return TAHOMA_TO_FAN_MODES[self.select_state(IO_AIR_DEMAND_MODE_STATE)]
+
+    @property
+    def fan_modes(self) -> Optional[List[str]]:
+        """Return the list of available fan modes."""
+        return [*FAN_MODES_TO_TAHOMA]
+
+    async def async_set_fan_mode(self, fan_mode: str) -> None:
+        """Set new target fan mode."""
+        if fan_mode == FAN_BYPASS:
+            await self.async_execute_command(COMMAND_SET_AIR_DEMAND_MODE, "auto")
```

Comment:
The command to set "cooling" is missing : 
`await self._set_ventilation_mode(cooling="on")`

Reply:
Yes it's needed to launch the cooling mode, otherwise how do you set cooling at "on"?
Before, it was set with HVAC_MODE but it's no longer the case.

Reply:
Why do you need to set cooling to 'on', if it is already on? Did you give this code a try and doesn't it work?

Reply:
It's not already "on", it's the FAN_MODE we want to set.

In your code, there is no command to set cooling="on" or cooling="off", so there is a problem!

Reply:
I don't know enough about this device, but the previous implementation didn't make sense to me. Why make it possible to enable it, when it is already on / off? This means that you

Shouldn't we add HVAC modes for cooling on / off?

Reply:
It's not a "real" cooling mode, it's just a bypass to not heat the air coming from outside with the air going outside from home.

The commands needed are in the issue #376 : 

Set "Cooling" mode :
```
2021-02-04 13:45:34 INFO (MainThread) [custom_components.tahoma] 2021-02-02 16:06:21: setAirDemandMode via Android 2.4.8#220 null on io://xxxx-xxxx-xxxx/15021786#1, with ['auto'].
2021-02-04 13:45:34 INFO (MainThread) [custom_components.tahoma] 2021-02-02 16:06:21: setVentilationMode executed via Android 2.4.8#220 null on io://xxxx-xxxx-xxxx/15021786#1, with [{'test': 'off', 'end_of_line_test': 'off', 'leap_year': 'off', 'prog': 'off', 'month': 2, 'day_night': 'night', 'cooling': 'on', 'day': 4}].
```
Stop "Cooling" mode :
```
2021-02-04 13:45:34 INFO (MainThread) [custom_components.tahoma] 2021-02-02 16:06:21: setVentilationMode executed via Android 2.4.8#220 null on io://xxxx-xxxx-xxxx/15021786#1, with [{'test': 'off', 'end_of_line_test': 'off', 'leap_year': 'off', 'prog': 'off', 'month': 2, 'day_night': 'night', 'cooling': 'off', 'day': 4}].
```